### PR TITLE
change described output html format to image

### PR DESCRIPTION
**Pull request by mhrimaz**
GitHub doesn't show the html styles correctly. I simply put images instead of them.

### CODE REVIEW

File: ADDONS.md [hunk 1]
```diff
@@ -46,13 +46,11 @@ Foreground and background colors do not replace one another but they sum up, whi
 
 So the following sequence, which is not terminated by reset, `\u001B[41m``AAA``\u001B[40m``\u001B[37m``BBB` produces as output:
 
-<span style="color:white;background-color:black; display:inline-block; width:100%"><span style="color:black;background-color:red;display:inline-block">AAA</span>BBB    
-</span>
+![withoutreset](colored-message-without-reseting.png "")
```
Comment (alessiogambi) [inline ADDONS.md:49]: I like the idea of using images, but this output is not 100% correct. The '`' characters belong to the Markdown format and not the output. The correct image is:
Screen Shot 2020-06-25 at 20 38 52

File: ADDONS.md [hunk 2]
```diff
@@ -46,13 +46,11 @@ Foreground and background colors do not replace one another but they sum up, whi
...
 
 While the following sequence, which includes the reset control character, `\u001B[41m``AAA``\u001B[40m``\u001B[37m``BBB``\u001B[0m` produces as output:
 
-<span style="color:black;background-color:red;display:inline-block">AAA</span><span style="color:white;background-color:black;display:inline-block">BBB</br>
-</span>
+![withreset](colored-message-with-reseting.png "")
```
Comment (alessiogambi) [inline ADDONS.md:59]: I like the idea of using images, but this output is not 100% correct. The '`' characters belong to the Markdown format and not the output. The correct image is:
Screen Shot 2020-06-25 at 20 39 54

**Review by alessiogambi — CHANGES_REQUESTED** [review]
Please update the images to match the expectations. I provided images in the comment, maybe you can use them?

**Comment by mhrimaz** [comment]
@alessiogambi How are the images this time? I also included the java code for further clarification.

**Review by alessiogambi — APPROVED** [review]
Looks good. Thanks

**Pull request merged**